src/util/redirects-raw.txt

```diff
@@ -0,0 +1,35 @@
+/docs/configurations/overview/                   /docs/configure/overview/                                                      301
```

Comment:
Questions for reviewers:

1. Is the name & location of this file correct?
2. I formatted the whitespace of this file (and `static/_redirects`) to make it easier to read. Is that alright?
3. To the best of my knowledge each of these redirects should include the framework in both the "from" and "to" paths. Is that correct?

Reply:
On (3), when looking at the table here: https://storybook.js.org/docs/react/faq#is-it-possible-to-browse-the-documentation-for-past-versions-of-storybook, since there are no entries that exist in pre-6.0 docs but not in 6+ docs, I _think_ all of the redirects here should have the framework included.

Reply:
3. Yes, include the framework in from & to for the post-6.0 docs. Any docs pages [after Aug 2020](https://github.com/storybookjs/frontpage/pull/130) have the framework in the URL